cwrc_preserver.rb

```diff
@@ -15,28 +19,57 @@ module CWRCPerserver
   # process command line arguments
```

Comment:
Wondering if some of these can also be set as default in the configuration file `secrets.yml`? like `debug`, and `reprocess`, and can be overwritten on the command line?
also, assume start_dt is the time stamp for the modified date? could you please add what format this value should be in? yyyy-mm-dd?

Reply:
Added `debug`. Improved documentation. `reprocess` likely only gets used if someone creates a list of IDs by hand so maybe best not adding to a config.